README.md

```diff
@@ -0,0 +1,64 @@
+# Breadcrumb
+Breadcrumb is a canonical component that any VTEX app can use.
+
+To use it in your code you should first add it as a dependencie into your app's ```manifest.json```, like:
+
+```json
+"dependencies": {
+    "vtex.breadcrumb": "0.x",
+    "vtex.category-menu": "0.x"
```

Comment:
remove this

Reply:
the category-menu or the whole dependencies ... ?

Reply:
category-menu

Reply:
👍